### PR TITLE
[Bug] generate exceptions to avoid mulitDistinctAggregation produce wrong result #3492

### PR DESCRIPTION
**Pull request by wutiangan**
when a query (#3492) contain “2 DistinctAggregation with one column” and “1 DistinctAggregation with two columns”,  it will produce wrong result.

This pull request is not to solve this problem really, but to generate exceptions to avoid getting wrong results. 

This problem needs a real repair in future.

### CODE REVIEW

File: fe/src/test/java/org/apache/doris/analysis/AggregateTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```
Comment (morningman) [inline fe/src/test/java/org/apache/doris/analysis/AggregateTest.java:1]: I think we can just move this test to `QueryPlanTest.java`

**Review by morningman — CHANGES_REQUESTED** [review]
Please rebase the code to make UT happy

File: fe/src/test/java/org/apache/doris/analysis/AggregateTest.java (hunk 1 above)
Comment (wutiangan) [reply fe/src/test/java/org/apache/doris/analysis/AggregateTest.java:1]: I think this is an exception in the analysis phase, not execution plan phase. 
This file is specially used for unit tests related to aggregate function analysis

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**